lib/src/ast/property.dart

```diff
@@ -127,13 +133,10 @@ class ParsedPropertyAst extends TemplateAst
     this.nameToken,
     this.suffixToken, [
     this.valueToken,
+    this.expression,
```

Comment:
+1 to this change.

I don't like that it used to fire a dart parser to create a node! I the expression being passed in